python/uv/__init__.py

```diff
@@ -41,8 +16,7 @@ def __getattr__(attr_name: str) -> object:
         "get_requires_for_build_editable",
         "prepare_metadata_for_build_editable",
     }:
-        err = f"Using `uv.{attr_name}` is not allowed. The uv build backend requires preview mode to be enabled, e.g., via the `UV_PREVIEW=1` environment variable."
+        err = (
+            f"Using `uv.{attr_name}` is not allowed. Consider using `uv_build` instead."
```

Comment:
Can we say a bit more here? Like. 
```suggestion
            f"Using `uv.{attr_name}` is not allowed; build backend functionality is in the `uv_build` package. Did you mean to use `uv_build` as your build system?"
```